WordPress/UITestsFoundation/Screens/Login/Unified/PasswordScreen.swift

```diff
@@ -48,12 +50,7 @@ public class PasswordScreen: ScreenObject {
         let continueButton = app.buttons["Continue Button"]
         continueButton.tap()
 
-        // The Simulator might ask to save the password which, of course, we don't want to do
-        if app.buttons["Save Password"].waitForExistence(timeout: 5) {
-            // There should be no need to wait for this button to exist since it's part of the same
-            // alert where "Save Password" is.
-            app.buttons["Not Now"].tap()
-        }
+        app.dismissSavePasswordPrompt()
```

Comment:
We are calling the new `dismissSavePasswordPrompt()` logic in three different places.

There might be a more centralized location for this. At the moment, I'm more interested in getting the builds to a stable place. We can refactor later.